plugins/vault-backend/README.md

```diff
@@ -83,6 +83,28 @@ To get started, first you need a running instance of Vault. You can follow [this
      }
    ```
 
+## Login via Kubernetes Auth Method
+
+Alternatively, it's also possible to make Backstage use the Kubernetes auth method to login into Vault. For that, the configuration should look like this:
```

Comment:
Please link to the upstream Vault docs for the "Kubernetes auth method" here, so people don't have to look up themselves what that is.

Reply:
Done. It was linked below, But done it here now